bucket/freerapid.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": "0.9u4",
+    "description": "A Java downloader that supports downloading from Rapidshare, Youtube, Facebook, Picasa and other file-sharing services.",
+    "homepage": "http://wordrider.net/freerapid/",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://wordrider.net/freerapid/faq.html"
+    },
+    "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/freerapid/FreeRapid-0.9u4.zip",
+    "hash": "f26d570b63e9591e438a625f87eee80480b160f077a4d7aae433c8d3e0d20ab9",
+    "suggest": {
+        "JAVA Runtime Environment": "java/openjdk"
+    },
+    "extract_dir": "FreeRapid-0.9u4",
+    "bin": "frd.jar",
+    "shortcuts": [
+        [
+            "frd.jar",
```

Comment:
This will require .jar extension to be registered to be handled by the user's JRE